main.go

```diff
@@ -111,7 +118,7 @@ func main() {
 	fmt.Printf("Initialized Turnkey client successfully. Turnkey API User UUID: %s\n", userID)
 
 	router.GET("/", func(ctx *gin.Context) {
-		ctx.String(http.StatusOK, "This is the Demo Passkey Wallet backend. Welcome I guess? Head to https://wallet.tx.xyz if you're lost.")
+		ctx.String(http.StatusOK, fmt.Sprintf("This is the Demo Passkey Wallet backend. Welcome I guess? Head to %s if you're lost.", allowedOrigins[0]))
```

Comment:
And here you should be able to use os.Getenv("CLIENT_ORIGIN") as well, no funny business with multiple elements in this array :)